src/timeMachine/components/FluxToolbar.tsx

```diff
@@ -30,17 +29,26 @@ const FluxToolbar: FC<Props> = ({onInsertFluxFunction, onInsertVariable}) => {
 
   let activeToolbar
 
+  const onInsertFluxFunction = (fn: FluxFunction | FluxToolbarFunction) => {
```

Comment:
moved in from parent component since we might as well reference the context where we use it